fpsim/locations/kenya/kenya_bf_stats.csv

```diff
@@ -0,0 +1,3 @@
+"","fitgumbel$estimate"
+"a",11.4261936291137
```

Comment:
This duration isn't what I would expect based on DHS stat compiler

Reply:
Very possible/likely that I'm misunderstanding the statistics here, just trying to make sense of why the mu dropped by half between Senegal and Kenya

Reply:
I'm not sure why they are showing up the exact same in statcompiler, other than possibly the weights (I haven't added weights to either calculation), or some handling of the zeros that the DHS is doing that we are not.